osu.Framework.Tests/Visual/Platform/TestSceneBorderless.cs

```diff
@@ -104,59 +57,27 @@ public TestSceneBorderless()
                     }
                 }
             };
-
-            windowMode.ValueChanged += newMode => currentWindowMode.Text = $"Window Mode: {newMode}";
-            windowMode.TriggerChange();
         }
 
-        private Container createScreen(Display display, string name) =>
-            new Container
-            {
-                X = display.Bounds.X,
-                Y = display.Bounds.Y,
-                Width = display.Bounds.Width,
-                Height = display.Bounds.Height,
-
-                BorderColour = display.Index == 0 ? active_stroke : screen_stroke,
-                BorderThickness = 20,
-                Masking = true,
-
-                Children = new Drawable[]
-                {
-                    new Box
-                    {
-                        RelativeSizeAxes = Axes.Both,
-                        Colour = display.Index == 0 ? active_fill : screen_fill
-                    },
-                    new SpriteText
-                    {
-                        Padding = new MarginPadding(50),
-                        Text = name,
-                        Font = new FontUsage(size: 200),
-                        Colour = Color4.Black
-                    }
-                }
-            };
-
         [BackgroundDependencyLoader]
         private void load(FrameworkConfigManager config, GameHost host)
         {
             window = host.Window as SDL2DesktopWindow;
             config.BindWith(FrameworkSetting.WindowMode, windowMode);
 
+            windowMode.BindValueChanged(mode => currentWindowMode.Text = $"Window Mode: {mode.NewValue}", true);
+
             if (window == null)
             {
-                Logger.Log("No suitable window found");
                 return;
             }
 
-            window.DisplaysChanged += onDisplaysChanged;
-            refreshScreens(window.Displays);
-
             const string desc2 = "Check whether the window size is one pixel wider than the screen in each direction";
 
             Point originalWindowPosition = Point.Empty;
 
+            AddStep("nothing", () => { });
```

Comment:
we probably want to change how test scenes should behave on first run, but at least add the comment so that this is not mistaken for leftover/debug code.
```suggestion
            // so the test case doesn't change fullscreen size just when you enter it
            AddStep("nothing", () => { });
```